pandas/core/computation/eval.py

```diff
@@ -193,6 +193,8 @@ def eval(
     corresponding bitwise operators.  :class:`~pandas.Series` and
     :class:`~pandas.DataFrame` objects are supported and behave as they would
     with plain ol' Python evaluation.
+    This allows `eval` to run arbitrary code, which can make you vulnerable to code
+    injection if you pass user input to this function.
```

Comment:
This is an internal method, right? In such case, I don't think this is necessary.

Reply:
Ah - this is exposed at the top level. Agree with including it, but I don't think starting with `This` is accurate - it 
 reads to me like that refers to the previous sentence. Can just say

> `eval` can run arbitrary code, ...